library/eggdrop

```diff
@@ -2,9 +2,11 @@ Maintainers: Geo Van O <geo@eggheads.org> (@vanosg)
 GitRepo: https://github.com/eggheads/eggdrop-docker.git
 
 Tags: develop
-GitCommit: 68e421d4e8f87301d921c84548407ec340f0b8ed
+Architecture: arm32v6, arm32v7, amd64
```

Comment:
This isn't quite right -- `eggdrop` is `FROM alpine:3.7`, which only supports `arm32v6` (not `v7`) -- the `v6` image should run fine on a `v7` system, but this list needs to be a subset of the parent. :+1:

Alpine 3.7's `Architectures:` list, for reference:

https://github.com/docker-library/official-images/blob/ab2071fa3424f94ebc9afaf157fdb7612a971b77/library/alpine#L30